### DISCUSSION TITLE
Vulnerability Management/Flag outdated software installed?

### DISCUSSION THREAD

**Discussion by luminous706** [amidaware/tacticalrmm#1332] [General]
Not sure if that's a feature or not already, but does Tactical flag the outdated software that are installed? Would be a great feature to have!

**Comment by wh1te909** [comment]
github dependabot takes care of that and will alert us if it finds any vulnerable packages
if you mean actual system updates on your linux server then no, you'll need to manage that
if you go with our hosted option then we fully manage that for you

**Reply by luminous706** [reply]
I meant the applications installed on a Windows endpoint, where we deploy the Tactical Agent. I was wondering if Tactical, since it will list the apps and versions, can also flag if let's say, Firefox is outdated, etc. Maybe the feature is already there and forgot about it, I haven't use Tactical recently, I was asking for a client who is interested.

**Reply by wh1te909** [reply]
LMAO yea totally misunderstood you then haha. ok so yea similar to https://github.com/amidaware/tacticalrmm/issues/1148 no there is no such feature atm

**Reply by luminous706** [reply]
no worries, your answer was still insightful, I didn't know about github dependabot, good to know!
